src/sql/workbench/browser/modelComponents/componentBase.ts

```diff
@@ -201,6 +201,14 @@ export abstract class ComponentBase<TPropertyBag extends azdata.ComponentPropert
 		this.setPropertyFromUI<boolean>((props, value) => props.ariaHidden = value, newValue);
 	}
 
+	public get ariaLive(): string | undefined {
```

Comment:
This doesn't really belong in the base class - most things aren't going to have an aria live property make sense so adding it to everything is a bit much.